x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.test.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MockRouter, mockDependencies } from '../../__mocks__';
+
+import { RequestHandlerContext } from '@kbn/core/server';
+
+jest.mock('../../lib/indices/fetch_ml_inference_pipeline_processors', () => ({
+  fetchMlInferencePipelineProcessors: jest.fn(),
```

Comment:
Ahhh I should've used this in my unit tests! Mocking the actual functions! Thanks for this.

Reply:
:) No worries... now, get back to your vacation 😂